problemsets-mat223/modules/module3-exercises.tex

```diff
@@ -16,7 +16,14 @@
 			\item Can $A$ be extended (i.e., can vectors be added to $A$) so that
 				$A$ spans all of $\R^3$?
 		\end{enumerate}
-
+		\begin{solution}
```

Comment:
I think I understand what you're doing wrong. You need to checkout `master` before you make a new branch, otherwise your new branch will have all the changes from the old branch *and* your new changes.

If you need to refenaggle everything the easiest way is to clone the base repo into a brand new directory, make a branch, copy the file over, and then commit. I usually find that easier than switching back and forth a bunch. Make sure to give your new new branches new names (e.g., `module6-updates`) so that github doesn't complain.